pkg/azurefile/controllerserver.go

```diff
@@ -104,6 +106,10 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	if acquired := d.volumeLocks.TryAcquire(volName); !acquired {
+		if req.GetVolumeContentSource() != nil {
+			_, err := getAzcopyJob(volName)
```

Comment:
what's the logic here? just want to provide the copy status here? but why there is a warning in L111?

Reply:
yes, the logic here is want to print copy percent in csi log before return `status.Errorf(codes.Aborted, volumeOperationAlreadyExistsFmt, volName)` in a new createvolume req, e.g.:
```
I0824 07:56:28.339951       1 utils.go:76] GRPC call: /csi.v1.Controller/CreateVolume
I0824 07:56:28.341278       1 utils.go:77] GRPC request: {"capacity_range":{"required_bytes":1084479242240},"name":"pvc-ba88fffa-c067-41f6-b5f8-c8f22e4a4d4c","parameters":{"csi.storage.k8s.io/pv/name":"pvc-ba88fffa-c067-41f6-b5f8-c8f22e4a4d4c","csi.storage.k8s.io/pvc/name":"pvc-azurefile-clone","csi.storage.k8s.io/pvc/namespace":"default","skuName":"Premium_LRS"},"volume_capabilities":[{"AccessType":{"Mount":{"mount_flags":["dir_mode=0777","file_mode=0777","mfsymlinks","cache=strict","nosharesock","actimeo=30"]}},"access_mode":{"mode":5}}],"volume_content_source":{"Type":{"Volume":{"volume_id":"mc_aks-clonetest_group_aks-clonetest_westus2#f1e4862d650b341cf991769#pvc-9d36661e-5e6f-466a-aba6-3a1284d7f8af###default"}}}}
I0824 07:56:28.434592       1 controllerserver.go:118] azcopy jobid c4fd9aea-f64a-914c-60a7-0dfd2890a078 is in progress, copy percent: 11.1
E0824 07:56:28.435680       1 utils.go:81] GRPC error: rpc error: code = Aborted desc = An operation with the given Volume ID pvc-ba88fffa-c067-41f6-b5f8-c8f22e4a4d4c already exists
```

Reply:
then this line is not needed?
```
klog.Warningf("getAzcopyJob failed in GetVolumeContentSource, err: %v", err)
```

Reply:
done, improve the logic here.